client/src/App.tsx

```diff
@@ -33,9 +33,9 @@ function App(): JSX.Element {
                   <Route exact path="/dashboard">
                     <Dashboard />
                   </Route>
-                  <Route path="*">
+                  {/* <Route path="*">
                     <Redirect to="/login" />
-                  </Route>
+                  </Route> */}
```

Comment:
Could we please try to avoid commenting out code that doesn't need to be commented out, this is just going to lead to problems in the future 😄. Thank you!

Reply:
Sorry. I always forget to uncommented this.